content/influxdb/cloud/organizations/buckets/_index.md

```diff
@@ -11,7 +11,7 @@ influxdb/cloud/tags: [buckets]
 ---
 
 A **bucket** is a named location where time series data is stored.
-All buckets have a **retention policy**, a duration of time that each data point persists.
+All buckets have a **retention period**, a duration of time that each data point persists.
```

Comment:
I suspect this might benefit from some clarification or detail. I can't think of a good suggestion for the text but I think it would be valuable for the reader to understand that the retention policy applies to the time specified by the _timestamps_ on data points, rather than the wall-clock time. For most users these are the same thing (they submit all their data with a timestamp of `now()` but other users do write data into the past or future. Maybe it's worth mentioning this edge case somewhere?

Reply:
I noticed in the Glossary you explain this concept. Maybe link the above `retention policy` to it's definition in the glossary?